index.js

```diff
@@ -0,0 +1,143 @@
+
+// If there's local data available,
+if (isStorageAvailable('localStorage')) {
+  const data = JSON.parse(localStorage.getItem('bookList'));
+  // and if it's not empty, update it
+  if (data) {
+    books = JSON.parse(localStorage.getItem('bookList'));
+  }
+}
+
+// Create classes
+class Bookshelf {
+  constructor(books) {
+    this.books = books;
+  }
+
+  addBook(title, author) {
+    const book = {
+      id: Math.floor(Math.random() * 1000000),
+      title: title.value,
+      author: author.value,
+    };
+
+    this.books.push(book);
+
+    if (isStorageAvailable('localStorage')) {
+      localStorage.setItem('bookList', JSON.stringify(this.books));
+    }
+  }
+
+  remove(id) {
+    this.books = this.books.filter((book) => book.id !== parseInt(id, 10));
+    localStorage.setItem('bookList', JSON.stringify(this.books));
+    this.updateBookList();
+  }
+
+  updateBookList() {
+    bookList.innerHTML = '';
+
+    this.books.forEach((el) => {
+      bookList.innerHTML += `<div>
+      <p>"<span class="title">${el.title}</span>" by <span class="author">${el.author}</span></p>
+      <button id="${el.title}" onclick="remove('${el.id}')" class="remove">Remove</button>
+      </div>`;
+    });
+  }
+}
+
+const newbook = new Bookshelf(books);
+
+form.onsubmit = () => {
+  newbook.addBook(title, author);
+
+  newbook.updateBookList();
+
+  form.reset();
+};
+
+newbook.updateBookList();
+
+const remove = (id) => {
+  newbook.remove(id);
+};
+remove();
+const allBooks = document.querySelector('.all-books');
+const addBook = document.querySelector('.add-book');
+const contact = document.querySelector('.contact');
+const navList = document.querySelector('#list');
+const navAdd = document.querySelector('#add');
+const navContact = document.querySelector('#contact');
+
+navList.onclick = () => {
+  navList.style.color = 'darkblue';
+  navAdd.style.removeProperty('color');
+  navContact.style.removeProperty('color');
+  allBooks.classList.remove('hide');
+  addBook.classList.add('hide');
+  contact.classList.add('hide');
+};
+
+navAdd.onclick = () => {
+  navAdd.style.color = 'darkblue';
+  navList.style.removeProperty('color');
+  navContact.style.removeProperty('color');
+  addBook.classList.remove('hide');
+  allBooks.classList.add('hide');
+  contact.classList.add('hide');
+};
+
+navContact.onclick = () => {
+  navContact.style.color = 'darkblue';
+  navList.style.removeProperty('color');
+  navAdd.style.removeProperty('color');
+  contact.classList.remove('hide');
+  addBook.classList.add('hide');
+  allBooks.classList.add('hide');
+};
+
+const siteDate = document.querySelector('#date');
+
+function time() {
+  const date = new Date();
+  const locale = navigator.language;
+  const options = {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: 'true',
+  };
+  siteDate.textContent = `${date.toLocaleTimeString(locale, options)}`;
+  return siteDate;
+}
```

Comment:
- [ ] Kindly make sure to display the current date using [Luxon](https://moment.github.io/luxon/#/install) as per the project requirements.
See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).
- [ ] Also make sure to align the date/time to the right as shown in the attached video as per the project requirements.

https://user-images.githubusercontent.com/70269507/165714564-a51ca3e3-e2a6-4db8-9a48-599a89e102a8.mov

Reply:
resolved